logos/logos-workernode/logos_worker_node/lane_manager.py

```
@@ -1297,8 +1303,18 @@ def _estimate_lane_vram_mb(self, lane_config: LaneConfig) -> float:
         # Calibrated base_residency_mb already includes KV (measured under the
         # configured cap; see calibration.py). Adding KV again double-counts.
         # "measured" source is weights-only — legacy add-KV stays correct there.
+        #
+        # BUT base_residency is frequently the calibration-time FULL GPU
+        # reservation (gpu_memory_utilization × device VRAM) — every model
```

Ruff (0.15.18)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)